### ISSUE TITLE
Docs is not supporting react native ios objective-c

### ISSUE THREAD

**Issue by areejmayadeh** [tiktok/tiktok-opensdk-ios#21]
Since react native IOS is written in Objective-C your SDK is not showing support for this case any help ?

**Comment by Nishannb** [comment]
Yes, I also wanted to implement Tiktok Share Kit SDK in my react native app but the docs seems to only have iOS SDK. Do we have react native SDK for ShareKit? @areejmayadeh did you found any solution ?

**Comment by Dokome** [comment]
You can refer to react-native-tiktok codebase and create the native module
